pkg/downwardmetrics/downwardmetrics.go

```diff
@@ -4,12 +4,19 @@ import (
 	"path/filepath"
 	"strconv"
 
+	"kubevirt.io/kubevirt/pkg/util"
+
 	v1 "kubevirt.io/api/core/v1"
 
 	"kubevirt.io/kubevirt/pkg/config"
 	"kubevirt.io/kubevirt/pkg/downwardmetrics/vhostmd"
 )
 
+var (
```

Comment:
Can you make this a const please?

Reply:
The function `filepath.Join()` does not allows me to const these values: `Const initializer 'filepath.Join(util.VirtPrivateDir, " downwardmetrics-channel")  is not a constant`. Constants cannot be set at run-time, only in compile time. I can do this with a concatenation like:

```go
const (
	DownwardMetricsChannelDir    = util.VirtPrivateDir + "/downwardmetrics-channel"
	DownwardMetricsChannelSocket = DownwardMetricsChannelDir + "/downwardmetrics.sock")
)
```
However, I'm not sure about the potential downsides of one approach or the other. `filepath.Join()` seems to be safer than a concatenation. Nevertheless, `util.VirtPrivateDir` is a hardcoded path, so I guess I does not really matters. WDYT?

Reply:
Done.